openbas-api/src/test/java/io/openbas/service/InjectServiceTest.java

```diff
@@ -250,11 +253,15 @@ void testImportXlsAbsoluteDate() throws IOException {
             when(userRepository.findById(any())).thenReturn(Optional.of(mockedUser));
             Team team1 = new Team();
             team1.setName("team1");
+            team1.setUsers(List.of(new User()));
```

Comment:
I am not sure how you cover the bug with this test

Reply:
That's a good point, this test will only test one of the two if fork from https://github.com/OpenBAS-Platform/openbas/pull/1477/files#diff-f41ecd0d8c78018477a00f131a786b67748d1421903f22a472a06fc9a563f20aR367
I added a test to test the other one so we should be better now.